PlaceRouteHierFlow/cap_placer/capplacer.cpp

```diff
@@ -419,8 +419,9 @@ fillContact (PnRDB::contact& con, const PnRDB::bbox& box) {
 class MinMax : public PnRDB::bbox {
 public:
     MinMax () : bbox (INT_MAX, INT_MAX, INT_MIN, INT_MIN){}
-    MinMax (const PnRDB::bbox &b) : bbox (b) {}
-    void update (const PnRDB::bbox& obox) { *this = unionBox(obox); }
+    explicit MinMax (const PnRDB::bbox &b) : bbox (b) {}
+    explicit MinMax (const MinMax &m) : bbox (m) {}
+    void update (const PnRDB::bbox& obox) { *this = MinMax(unionBox(obox)); }
```

Comment:
Do you want to disallow an operator= for this class?
Could make it private and not implement it?

Reply:
Not sure.   I mostly want to update it with a bbox.  Assignment should be fine.  Did you see a problem with allowing for the default assignment operator to be generated?  I had to create the constructor as a constructor with bbox needs an overriding one for MinMax itself.